README.md

```diff
@@ -203,6 +204,96 @@ You need to add any referrer in mpv by opening mpv [(playstore version)](https:/
 referrer="https://animixplay.to/"
 ```
 
+### Steam Deck
+
+#### Copypaste script:
+
+* Switch to Desktop mode (`STEAM` Button > Power > Switch to Desktop)
+* Open `Konsole` (Steam Deck Icon in bottom left corner > System > Konsole)
+* Copy the script, paste it in the CLI and press Enter("A" button on Steam Deck) 
+
+```
+mkdir ~/.local/bin
+echo "export $PATH=$HOME/.local/bin:$PATH" >> ".$(echo $SHELL | cut -d '/' -f 3)rc" 
+
+git clone --depth 1 https://github.com/junegunn/fzf.git ~/.fzf
+~/.fzf/install
+
+mkdir ~/.aria2c
+wget -O ~/.aria2c/aria2-1.36.0.tar.bz2 https://github.com/q3aql/aria2-static-builds/releases/download/v1.36.0/aria2-1.36.0-linux-gnu-64bit-build1.tar.bz2
+tar xvf ~/.aria2c/aria2-1.36.0.tar.bz2 -C ~/.aria2c/
+cp ~/.aria2c/aria2-1.36.0-linux-gnu-64bit-build1/aria2c ~/.local/bin/
+chmod +x ~/.local/bin/aria2c
+
+git clone https://github.com/pystardust/ani-cli.git ~/.ani-cli
+cd ~/.ani-cli && git checkout v4 && cd ~ 
+cp ~/.ani-cli/ani-cli ~/.local/bin/
+chmod +x ~/.local/bin/ani-cli
+
+flatpak install io.mpv.Mpv
+```
+press enter("A" button on Steam Deck) on questions
+
+#### Installation in steps:
+
+##### Install mpv (Flatpak version):
+
+```
+flatpak install io.mpv.Mpv
+```
+press enter("A" button on Steam Deck) on questions
+
+##### Install [fzf](https://github.com/junegunn/fzf): 
+
+```
+git clone --depth 1 https://github.com/junegunn/fzf.git ~/.fzf
+~/.fzf/install
+```
+press enter("A" button on Steam Deck) on questions
+
+##### Make a ~/.local/bin folder if doesnt exist and add it to $PATH
+
+```
+mkdir ~/.local/bin
+echo "export $PATH=$HOME/.local/bin:$PATH" >> ".$(echo $SHELL | cut -d '/' -f 3)rc" 
+```
+
+##### Install [aria2](https://github.com/aria2/aria2) (needed for download feature only):
+
+```
+mkdir ~/.aria2c
+wget -O ~/.aria2c/aria2-1.36.0.tar.bz2 https://github.com/q3aql/aria2-static-builds/releases/download/v1.36.0/aria2-1.36.0-linux-gnu-64bit-build1.tar.bz2
+tar xvf ~/.aria2c/aria2-1.36.0.tar.bz2 -C ~/.aria2c/
+cp ~/.aria2c/aria2-1.36.0-linux-gnu-64bit-build1/aria2c ~/.local/bin/
+chmod +x ~/.local/bin/aria2c
+```
+
+##### Install ani-cli:
+
+```
+git clone https://github.com/pystardust/ani-cli.git ~/.ani-cli
+cd ~/.ani-cli && git checkout v4 && cd ~ 
```

Comment:
`cd` is enough, no need for `cd ~`

Reply:
https://github.com/pystardust/ani-cli/pull/976/commits/c099e3c7a497f834c9f3cce110339f1f76219860